src/Material.cpp

```diff
@@ -816,7 +951,7 @@ void Material::setSigmaS(double* xs, int num_groups_squared) {
 
   for (int dest=0; dest < _num_groups; dest++) {
     for (int orig=0; orig < _num_groups; orig++)
-      _sigma_s[dest*_num_groups+orig] = xs[orig*_num_groups+dest];
+      setSigmaSByGroupInline(xs[orig*_num_groups+dest],orig,dest);
```

Comment:
No, there wasn't.  I can get rid of it.  The inline certainly isn't needed.  I was just trying to avoid putting the logic in too many places.  But I can call the non-inline version.